fabric-windows-example/patches/react-native-windows+0.74.11.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/react-native-windows/Microsoft.ReactNative/Modules/NativeUIManager.cpp b/node_modules/react-native-windows/Microsoft.ReactNative/Modules/NativeUIManager.cpp
+index 97c1691..8136008 100644
+--- a/node_modules/react-native-windows/Microsoft.ReactNative/Modules/NativeUIManager.cpp
++++ b/node_modules/react-native-windows/Microsoft.ReactNative/Modules/NativeUIManager.cpp
+@@ -309,16 +309,10 @@ static YGValue YGValueOrDefault(
+       return YGValue{static_cast<float>(pct.asDouble()), YGUnitPercent};
+     }
+     if (str.length() > 2 && (str.compare(str.length() - 2, 2, "pt") || str.compare(str.length() - 2, 2, "px"))) {
+-      shadowNode.RedBox(
```

Comment:
Just curious, will we be able to remove this patch in the future?

Reply:
This isn't actually needed for the fabric app. Thanks for catching it - will remove it.